provisioning/miniprov/hare_mp/main.py

```diff
@@ -378,7 +380,17 @@ def start(args):
         start_hax_and_consul_without_systemd(url, utils)
 
 
-def start_mkfs(hostname: str, hare_config_dir: str):
+def start_mkfs(cmd_line: List[str]) -> int:
+    try:
+        execute(cmd_line)
+        rc = 0
+    except Exception as error:
+        logging.error('Error during mkfs start (%s)', error)
```

Comment:
It is more beneficial if the error describes which mkfs process actually failed (at least, which fid was it).

Reply:
Addressed.